### ISSUE TITLE
save.Video silently fails to overwrite old file

### ISSUE THREAD

**Issue by maxheld83** [yihui/animation#60]
When I run `saveVideo()` again and again (with changes), the video file opens, but is not changed on the disk.
I am guessing it simply does not overwrite the old file of the same name.

The function issues no warning, but says:

```
Video has been created at: /Users/Max/Github/ODCD-experiments/animation.mp4
```

Can other people reproduce this behavior?

I am on:
- OS X 10.10.3
- Animation 2.3

**Comment by yulijia** [comment]
Hello Maximilian,

The development version of this package already fix this bug. We will update it to CRAN when we finish some test for other functions.

To install the GitHub version under development:

```
install.packages('animation', repos = 'http://rforge.net', type = 'source')
```

**Comment by flying-sheep** [comment]
i had the same problem and it took me some time to realize that ffmpeg isn’t steadfastly refusing to change the framerate, but it’s a bug in this package :unamused:

**Comment by yihui** [comment]
Should be fixed via 77af664f824f9af092c4bdd2d50d1418e9f5c670. I'm preparing a CRAN release now.
